process/transaction/transactionCostEstimator.go

```diff
@@ -90,11 +91,13 @@ func (tce *transactionCostEstimator) ComputeTransactionGasLimit(tx *transaction.
 		return tce.computeScCallGasLimit(tx)
 	case process.BuiltInFunctionCall:
 		return tce.computeScCallGasLimit(tx)
+	case process.RelayedTx:
+		// TODO for relayed txs not working yet
```

Comment:
Maybe rephrase the TODO? Something doesn't sound well :D
Add a Jira task for this?

Reply:
EN-9040 was created for this.